code/datums/wires/airlock.dm

```diff
@@ -115,7 +115,6 @@ var/const/AIRLOCK_WIRE_LIGHT = 2048
 				//Cutting this wire electrifies the door, so that the next person to touch the door without insulated gloves gets electrocuted.
 				if(A.secondsElectrified != -1)
 					A.shockedby += text("\[[time_stamp()]\][usr](ckey:[usr.ckey])")
-					add_logs(usr, A, "electrified", admin=0, addition="at [A.x],[A.y],[A.z]")
```

Comment:
This was broken and didn't work because add_logs( is specifically intended to be used on mobs